cggmp24-keygen/CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
-# Changelog
+# CGGMP24 releases
+
+## v0.6.0
+Update key generation protocol to follow the latest CGGMP24 paper revision.
+
+# CGGMP21 releases
 
 ## v0.5.0
 * Update `hd-wallet` dep to v0.6 [#120]
```

Comment:
Isn't first release of cggmp24 the 0.**7**.0 release? It's mentioned above in the migration guide and in our blog post

Reply:
Oh it's because I'm looking at keygen changlog, not the signing

Reply:
generally, cggmp24-keygen and cggmp24 don't have to follow the same versions, but I think for the first release we can sync them

Reply:
I don't actually think we should sync them here. I just didn't notice what I was commenting on